client/src/components/MultiJurisdictionAudit/Setup/Review/index.test.tsx

```diff
@@ -246,4 +246,48 @@ describe('Audit Setup > Review & Launch', () => {
       })
     })
   })
+
+  it.only('accepts custom sample size', async () => {
+    apiMock.mockImplementation(
+      generateApiMock(
+        sampleSizeMock,
+        contestMocks.filledTargetedWithJurisdictionId,
+        { jurisdictions: jurisdictionMocks.allManifests }
+      )
+    )
+    const { getByText, findByText, getAllByText } = render(
+      <Router>
+        <Review locked={false} prevStage={prevStage} refresh={jest.fn()} />
+      </Router>
+    )
+    const newSampleSize = await findByText(
+      'Enter your own sample size (not recommended)'
+    )
+    fireEvent.click(newSampleSize, { bubbles: true })
+    const customSampleSizeInput = await screen.findByRole('textbox')
```

Comment:
any reason you're using `screen` here but not in the rest of the test?

Reply:
@jonahkagan I was just adding the new test in, without updating the others yet, so I used the new features for the new test. I could probably easily swap out that bit, though.